pyproject.toml

```diff
@@ -16,6 +16,7 @@ pytest-asyncio = "^0.19.0"
 telliot-core = {git = "https://github.com/tellor-io/telliot-core.git", rev = "main"}
 telliot-feeds = {git = "https://github.com/tellor-io/telliot-feeds.git", rev = "main"}
 click = "^8.1.3"
+pydantic = "^1.10.2"
```

Comment:
This dep is getting removed from telliot-core/feeds in next release just fyi

Reply:
thanks